### DISCUSSION TITLE
Why tailwindcss is added in devDependencies in with-tailwindcss examples

### DISCUSSION THREAD

**Discussion by moshfiqrony** [vercel/next.js#28080] [Help] [answered]
Why this example below include tailwindcss, postcss, and autoprefixer as the devDependencis?
https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss
```
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "autoprefixer": "^10.2.6",
    "postcss": "^8.3.5",
    "tailwindcss": "^2.2.4"
  }
}
```

**Comment by jamesmosier** [comment, marked as answer]
Upon building your application, Tailwind styles are built into the code that is generated by running `next build`. Therefore there is no runtime dependency on `autoprefixer`, `postcss`, or `tailwindcss` and therefore they can safely be put in `devDependencies`.

**Reply by moshfiqrony** [reply]
Thanks.
